### ISSUE TITLE
`nix run .#regenerate-files` fails (missing `.attrs.sh`)

### ISSUE THREAD

**Issue by tripplilley** [nickel-lang/organist#200]
```
$ nix run .#regenerate-files
error: builder for '/nix/store/0gh34k924ii1n986w0m5sacq9430i7yv-regenerate-files.drv' failed with exit code 1;
       last 1 log lines:
       > bash: line 1: /tmp/nix-build-regenerate-files.drv-0/.attrs.sh: No such file or directory
       For full logs, run 'nix log /nix/store/0gh34k924ii1n986w0m5sacq9430i7yv-regenerate-files.drv'.
```

I'm still relatively new to Nix, Nickel, and Organist, so I fully allow this might be operator error.

My _project.ncl_ is bone stock output from `nix flake init -t github:nickel-lang/organist` and I _have_ done `nix run .#regenerate-lockfile`. I've tried this both from within and without a `nix develop` shell.

```
let inputs = import "./nickel.lock.ncl" in
let organist = inputs.organist in

{
  shells = organist.shells.Bash,

  shells.build = {
    packages = {},
  },

  shells.dev = {
    packages.hello = organist.import_nix "nixpkgs#hello",
  },
}
  | organist.OrganistExpression
```

**Comment by yannham** [comment]
Hi,

Thanks for the report. I just tried to copy the template  from the repo and I can `regenerate-lockfile` and `regenerate-files` both several times, from inside and outside the developer environment.

I think `.attrs.sh` should be provisioned by Nix automatically in our case (when using `__structuredAttrs`). I haven't seen this error before, personally.

Could you please let us know:

1. Your nix version (`nix --version`)
2. The organist version (e.g. the rev  written in the `flake.lock` file)
3. Your platform

**Comment by yannham** [comment]
After a quick search, it seems that organist is making use of `__structuredAttrs`, which only appears in the documentation from version 2.14 and onward (it's absent from [2.13 reference](https://nix.dev/manual/nix/2.13/language/advanced-attributes), but is there on the [2.14 reference](https://nix.dev/manual/nix/2.14/language/advanced-attributes)).

It should be clearly documented in the README though, so keeping this issue open until then. Is updating an option for you?

**Comment by tripplilley** [comment]
Yes, I just (finally) updated and it works fine now :sweat_smile: . Thanks!

```
$ nix --version
nix (Nix) 2.18.4
```

**Comment by tripplilley** [comment]
(oops, sorry - I closed it because it's "fixed", but then I saw the "keeping this issue open until then" note and re-opened it)
